Attribution Reporting.md

```diff
@@ -0,0 +1,187 @@
+### Default Value:
+Event source conversion reports are sent by the browser to the ad tech approximately one hour after the event’s expiry.  The default value is 30 days after registration.
+
+Navigation source conversion reports are sent following a more complex schedule of attribution windows. After the navigation, a schedule of built-in reporting windows begins.  By default, navigation reports are sent during one of three windows: 2, 7, or 30 days after registration.  Conversions registered before that deadline are sent at the end of that window.
+
+### Customizable Values:
+There are two values that can be used to customize report transmission times available as fields within the Attribution-Reporting-Register-Source header: report windows and expiration values.
+#### Report Windows:
+Report windows designate  how often reports will be sent.  The fields for customizing these values are:
+- event_report_window: A string representing a time in seconds, after which subsequent triggers won't be attributable to this source for the purpose of producing event-level reports (this is called a report window). If not set, the event report window falls back to the "expiry" value.
+- event_report_windows: An object representing a series of report windows, starting at "start_time", with reports for this source being delivered after each specified end time in "end_times". This can be used to vary the time of report delivery across multiple reports. If not set, the event report window falls back to the "expiry" value. Properties are as follows:
+- "start_time" Optional: A non-negative number specifying the start time for the reporting windows. If not specified, it defaults to 0.
+- "end_times": An array of positive numbers specifying end times for subsequent report windows. The values must be increasing, and greater than "start_time".
+    
+Only one of the fields,  "event_report_window" or "event_report_windows", can be specified.  If both are specified, the source registration will fail.
+
+#### Expiration Values:
+The expiration value designates the time after registration that the attribution source is active. This value defaults to 30 days. The field for customizing this value is:
+
+- expiry: A string representing an expiry time in seconds for the attribution source.
+The soonest value permitted for report windows and expiration values is 1 day. The latest period permitted is 30 days.
+
+### Reporting Delays
+Reports may not be sent exactly at the scheduled dates and times. If the browser isn't running when a report is scheduled to be sent, the report is sent at browser startup. The browser may also decide to delay some of these reports for a random, but short, duration. This duration should be no more than an hour.
+
+## Attribution Reports Data
+To send a report, the browser will make a non-credentialed (i.e. without session cookies) secure HTTP POST request to:
```

Comment:
Nit: saying "HTTPS post request" would be more clear than "secure HTTP POST request"